Data/Localizations/ru.lua

```diff
@@ -195,23 +193,20 @@ L.GDKP_AUCTION_DETAILS_GOLD_ADDED = [[
 |c00%s%sg добавлен в банк пользователем %s
 Примечания]];
 L.GDKP_AUCTION_DETAILS_GOLD_PAID_BY = [[
-%s заплатил |c00%s%sg за
-%s]];
+%s |cFFFFFFFFзаплатил |c00%s%sg |cFFFFFFFFза
```

Comment:
Stick to changing text only please. If a color change is in order somewhere then let me know and I'll fix it for all translations.